ztp/gitops-subscriptions/argocd/example/acmpolicygenerator/README.md

```diff
@@ -1,5 +1,78 @@
-## ACM PolicyGenerator
-The [ACM PolicyGenerator](https://github.com/stolostron/policy-generator-plugin) examples under [here](https://github.com/openshift-kni/cnf-features-deploy/tree/master/ztp/gitops-subscriptions/argocd/example/acmpolicygenerator) are defining the DU profile policies using [ACM PolicyGenerator reference API](https://github.com/stolostron/policy-generator-plugin/blob/main/docs/policygenerator-reference.yaml). These examples will generate ACM policies same as the DU profile policies generated from [policygentemplates example](https://github.com/openshift-kni/cnf-features-deploy/tree/master/ztp/gitops-subscriptions/argocd/example/policygentemplates) AND specifically the policy content object-definition (source-crs) are identical.
+# ACM PolicyGenerator 
+The ACM Policy Generator Custom Resource (CRs) is the official template mechanism used to generate ACM policies. It is replacing Policy Generator Templates or PGT.
+ACM Policy Generator CRs are similar to PGT but there are few notable differences regarding its patching and placement strategies.
+
+The [policy-generator-plugin](https://github.com/stolostron/policy-generator-plugin/policy-generator-plugin) examples in this directory are defining the DU profile policies using [ACM PolicyGenerator reference API](https://github.com/stolostron/policy-generator-plugin/policy-generator-plugin/blob/main/docs/policygenerator-reference.yaml). 
+
+These examples will generate ACM policies same as the DU profile policies generated from [policygentemplates example](https://github.com/openshift-kni/cnf-features-deploy/tree/master/ztp/gitops-subscriptions/argocd/example/policygentemplates) AND specifically the policy content object-definition (source-crs) are identical.
+
+# Comparison between PGT and ACM Policy Generator patching strategies
+
+| PGT patch | ACM Generator patch |
+|-----------|---------------------|
+| Uses Strategic merge [link](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-api-machinery/strategic-merge-patch.md)|PGT patches work by replacing variables with their values as defined by the patch|
```

Comment:
I'm wondering if it would be better to link to the kustomize documentation instead.
https://github.com/kubernetes-sigs/kustomize/blob/master/examples/patchMultipleObjects.md

Reply:
fixed